tox.ini

```diff
@@ -1,14 +1,15 @@
 [flake8]
 max-line-length=120
-per-file-ignores=__init__.py:F401,tests/*:S,D100,D104
+per-file-ignores=__init__.py:F401,tests/*:S, D,ANN
 docstring-convention=all
 import-order-style=pycharm
 application_import_names=api,tests
 exclude=gunicorn.conf.py
 ignore=
   # black compatibility:
   E203
-
+  # FastAPI dependencies
+  B008
```

Comment:
this should not be excluded from the entire source, IMO. Just where its used should have # noqa

Reply:
We might use FastAPI dependencies all around the code, it would be a burden to do it in the repository everywhere we use them.